### ISSUE TITLE
[Bug]: While navigating to “True/False” button in controls pane narrator announcing as true false check box checked which is incorrect.

### ISSUE THREAD

**Issue by msftedad** [storybookjs/storybook#24128]
### Describe the bug

While navigating to “True/False” button in controls pane narrator announcing as true false check box checked which is incorrect.

Actual Result:
While navigating to “True/False” button in controls pane narrator announcing as true false check box checked which is incorrect.

Expected Result:
While navigating to “True/False” button narrator should announce true checked/unchecked or false checked/unchecked.

### To Reproduce

                     
	
	
 1. Open URL: [Common Navigation - Scrolling Panel ⋅ Storybook (windows.net)](https://vivacommonstorybooktest.blob.core.windows.net/$web/index.html?path=/story/common-navigation--scrolling-panel&globals=backgrounds.grid:!false;backgrounds.value:transparent) 
2. Navigate to “App bar” button at right side of the pane invoke it and navigate to “Default” link invoke it. 
3. Now navigate to “Controls” tab item invoke it and navigate to “True/False” button and observe the narrator announcement.

Attachment:

https://github.com/storybookjs/storybook/assets/93735775/a38273ab-2032-48f3-bc50-f0456d95977b

### System

```bash
OS: Windows 11
OS version: 22H2 (OS Build 22621.2134)
Edge Version: 116.0.1938.69 (Official build) (64-bit)
```

**Comment by nishitiwari22** [comment]
Is this issue still open? I am looking to contribute to it.

**Comment by danielmarcano** [comment]
Hi, everyone. Hope you are all doing great. I have opened a [PR](https://github.com/storybookjs/storybook/pull/24418) solving this issue.
